nilearn/datasets/_utils.py

```diff
@@ -702,24 +702,24 @@ def movetree(src, dst):
 
     Any existing file is overwritten.
     """
-    names = os.listdir(src)
```

Comment:
I am wondering if we even need this movetree function.

Not sure how it differs
https://docs.python.org/3/library/shutil.html#shutil.move

@hndgzkn can you double check and maybe update the doc string to explain why it is different?

Reply:
thanks a bunch: I and future maintainers thanks you for the clarification.

Reply:
I hope it is clear enough.